app/main/views/services.py

```diff
@@ -157,6 +157,17 @@ def update_section(service_id, section):
                 "user",
                 "supplier app")
         except HTTPError as e:
+            errors_map = {}
+            for error in e.response.json()['error'].keys():
+                id = existing_service_content.get_question(error)['id']
+                errors_map[id] = {
+                    'input_name': error,
+                    'question': existing_service_content
+                    .get_question(error)['question'],
```

Comment:
It's a nitpick, but could we indent the `.get_question` lines so they stand out among the keys.

Side note: I think it's another example of how our line length code style check often doesn't make things better. Maybe it's worth discussing the idea of removing it.